react/components/LoginAsCustomer.tsx

```diff
@@ -35,12 +35,23 @@ export default class LoginAsCustomer extends Component<Props> {
 
     return (
       <div className={`${telemarketing.login}`}>
```

Comment:
Pretty sure `className={telemarketing.login}` is enough here.

Reply:
will change on #35